bl/files/files_test.go

```diff
@@ -77,14 +77,15 @@ func test_directory_file() *toAbsolutePathsTestCase {
 }
 
 func TestExtractYamlFileToUnknownStruct(t *testing.T) {
+	filesExtractor := FilesExtractor{}
 	t.Run("valid yaml file, should return an unknown struct and no error", func(t *testing.T) {
-		actualResult, actualErr := ExtractYamlFileToUnknownStruct("../../internal/fixtures/policyAsCode/valid-schema.yaml")
```

Comment:
I see no value for mocking the file extractor in this test, therefore changed to the real implementation :)